storm-core/src/jvm/org/apache/storm/metric/LoggingClusterMetricsConsumer.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.storm.metric;
+
+import org.apache.storm.metric.api.DataPoint;
+import org.apache.storm.metric.api.IClusterMetricsConsumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collection;
+
+/**
+ * Listens for cluster related metrics, dumps them to log
+ *
+ * To use, edit the storm.yaml config file:
+ *
+ * ```yaml
+ *   storm.cluster.metrics.register:
+ *     - class: "org.apache.storm.metrics.LoggingClusterMetricsConsumer"
+ * ```
+ *
+ */
+public class LoggingClusterMetricsConsumer implements IClusterMetricsConsumer {
+    public static final Logger LOG = LoggerFactory.getLogger(LoggingClusterMetricsConsumer.class);
+
+    static private String padding = "                       ";
+
+    @Override
+    public void prepare(Object registrationArgument) {
```

Comment:
This configuration is from yaml, same as MetricsConsumer, so it can be String, List of String, Map<String, Object>, and so on.

I don't like this way but having similar interface with IMetricsConsumer would be easier to use.